UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -584,8 +596,9 @@ private static bool IsPrimitive(GameObject gameObject)
 				&& gameObject.transform.localPosition == Vector3.zero
 				&& gameObject.transform.localRotation == Quaternion.identity
 				&& gameObject.transform.localScale == Vector3.one
-				&& gameObject.GetComponent<MeshFilter>() != null
-				&& gameObject.GetComponent<MeshRenderer>() != null;
+				&& 
+				((gameObject.GetComponent<MeshFilter>() != null && gameObject.GetComponent<MeshRenderer>() != null) || gameObject.GetComponent<SkinnedMeshRenderer>() != null);
```

Comment:
Since this logic is a bit long and difficult to parse, and since you have the same logic above in FilterPrimitives(), I suggest you create a function for it.  It will make these lines easier to read.  Something like:

private static bool ContainsValidRenderer(Gameobject gameobject)